src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -45,10 +45,13 @@ import scala.util.control.NonFatal
 
 object Boot extends IOApp with LazyLogging {
   val sentryDsn: Option[String] = sys.env.get("SENTRY_DSN")
+  val version: Option[String] = Option(getClass.getPackage.getImplementationVersion)
```

Comment:
Quick googling shows this returning a value set in the manifest of the jar.  I'm assuming that for us, this value is a commit hash?  I have no idea.  Did you test how this gets set and what types of values are there?  FWIW, as part of IndependiSam, we want to cut over to using semantic versioning, in which case you may be able to just read an ENV variable to get the version string.  This is trivial enough that we can probably keep it as is, we just should remember to come back and change this when we start semantic versioning.  Perhaps drop a comment here pointing at https://broadworkbench.atlassian.net/browse/ID-485 and make a comment in the ticket to come fix this line.

Reply:
I added a helpful comment thanks for pointing this out